scripts/zkai_d128_aggregated_proof_object_feasibility_gate.py

```
@@ -0,0 +1,1249 @@
+def relative_path(path: pathlib.Path | str) -> str:
+    return pathlib.Path(path).resolve().relative_to(ROOT.resolve()).as_posix()
+
+
+def expected_mutation_inventory() -> list[dict[str, Any]]:
+    return [
+        {"index": index, "mutation": mutation, "surface": surface}
+        for index, (mutation, surface) in enumerate(EXPECTED_MUTATION_INVENTORY)
+    ]
+
+
+def _validate_checked_block_receipt_payload(payload: Any) -> dict[str, Any]:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `payload`

(ANN401)